inspirehep/dojson/hep/fields/bd3xx.py

```diff
@@ -29,16 +29,16 @@
 from ..model import hep, hep2marc
 
 
-@hep.over('page_nr', '^300..')
+@hep.over('number_of_pages', '^300..')
 @utils.for_each_value
-def page_nr(self, key, value):
+def number_of_pages(self, key, value):
```

Comment:
You can remove the `for_each_value` because the element in the schema is now a single integer. Otherwise dojson would create a list of integers.